src/cyrillux.js

```diff
@@ -16,22 +16,25 @@ const cyrillicTrans = 'CYRILLIC <TRANS>';
 const sfs4900Trans = 'SFS4900 <TRANS>';
 
 export default function (config = {}) {
+  // console.log(`CONFIG=${JSON.stringify(config)}`); // eslint-disable-line no-console
 
```

Comment:
If config preprocessing is done here it needs to be done just once since config is given to "constructor". 
`const processedConf = preProcessConfig(config);`